web/src/App.tsx

```diff
@@ -1,32 +1,46 @@
-import React from 'react'
-import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import React from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
-import GamesList from './views/GamesList'
-import Game from './views/Game'
-import GameEdit from './views/Game/Edit'
+import GamesList from "./views/GamesList";
+import Game from "./views/Game";
+import GameEdit from "./views/Game/Edit";
 
-import StatsPage from './views/StatsPage'
-import Leaderboards from './views/Leaderboards'
-import ComparePlayers from './views/ComparePlayers'
+import StatsPage from "./views/StatsPage";
+import Leaderboards from "./views/Leaderboards";
+import ComparePlayers from "./views/ComparePlayers";
 
-import SalaryPage from './views/SalaryPage'
-import StatsTable from './views/StatsTable'
-import TeamDashboard from './views/TeamDashboard'
+import SalaryPage from "./views/SalaryPage";
+import StatsTable from "./views/StatsTable";
+import TeamDashboard from "./views/TeamDashboard";
+import { LeaderboardsContextProvider } from "./views/Leaderboards/LeaderboardsContext";
```

Comment:
It looks your editor is running prettier by default with different settings/style than the existing code. This makes it harder than it needs to be to review changes because of all the noise. Could you re-work this to make it easier for me to review? This PR should include only functional changes.

If you want to add standard prettier here I am okay with that but lets do it in a separate PR

Reply:
is there a prettier ruleset we are using that I can easily plug in?

Reply:
I don't think so (if there is nothing in the repo then no for sure). I think it just somewhat consistent because I wrote most of it in the same style. Standard prettier would be nice to add but we should do all the reformatting in a separate PR.